src/ReportGenerator.Core/Reporting/Builders/Rendering/HtmlRenderer.cs

```diff
@@ -1578,24 +1582,25 @@ private void SaveCss(string targetDirectory)
 
             using (var fs = new FileStream(targetPath, FileMode.Create))
```

Comment:
FileStream `fs` is never used. Therefore the CSS file is empty and the report is rendered without CSS.

Reply:
Good spots. Fixes pushed.